### PR TITLE
interfaces: allow unity7 AppMenu (Closes #1581191)

### PR DESCRIPTION
**Pull request by jdstrand**

### CODE REVIEW

File: interfaces/builtin/unity7.go [hunk 1]
```diff
@@ -117,41 +117,189 @@ dbus (receive, send)
...
+
+
 # Lttng tracing is very noisy and should not be allowed by confined apps. Can
 # safely deny. LP: #1260491
 deny /{,var/}run/shm/lttng-ust-* r,
 
-
-# TODO: pull in modern items from ubuntu-unity7-base abstraction, eg, HUD,
-# freedesktop notifications, etc
+# TODO: pull in modern items from ubuntu-unity7-base abstraction, eg, HUD, etc
```
Comment (3v1n0) [inline interfaces/builtin/unity7.go]: This should give you for free HUD items as well, so I guess you can remove it from list.

Comment (jdstrand) [inline interfaces/builtin/unity7.go]: Removed comment.

**Comment by mvo5** [comment]
:+1:

**Comment by mvo5** [comment]
retest this please

**Pull request merged**